src/python/fuzzing/fuzzer_selection.py

```diff
@@ -62,6 +62,37 @@ def update_mappings_for_fuzzer(fuzzer, mappings=None):
   ndb_utils.delete_multi([m.key for m in list(old_mappings.values())])
 
 
+def update_mappings_for_job(job, mappings):
+  """Clear existing mappings for a job, and replace them."""
+
+  old_fuzzers = data_types.Fuzzer.query(data_types.Fuzzer.jobs == job.name)
+  old_mappings = {}
+  for fuzzer in old_fuzzers:
+    old_mappings[fuzzer.name] = fuzzer
```

Comment:
We can do this:

```python
existing_fuzzers = {
  fuzzer.name: fuzzer for fuzzer in existing_fuzzers_query
}
```

to have a more declarative style that's easier to read.

Reply:
done, just added everything in a single line to avoid lint errors.